### ISSUE TITLE
Replace Gulp with something more updated, gulp relies on to many outdated packages

### ISSUE THREAD

**Issue by frjo** [HyphaApp/hypha#2689]
Gulp has not been updated in three years and in nom land that is an eternity. I think we should replace it to clear out a lot of old dependencies and minor security issues.

Gulp is used to:

* Build CSS and JS for production and CI.
* Watch CSS and JS during development.
* Build React app via Webpack
* Analyse React app via Webpack
* Copy fonts and images for production.
* Run Djangos collectstatic.

All source files are in `/hypha/static_src/` and Gulp put the resulting files in `/hypha/static_compiled/`.

Djangos `collectstatic` then looks in `/hypha/static_compiled/` since it is added to `STATICFILES_DIRS`.

**Comment by frjo** [comment]
I have already replaced Gulp with plain npm script for linting CSS and JS.

**Comment by frjo** [comment]
@vimal1083 What are the good and modern options for this? Webpack for everything, plain npm scripts or some other nice solution?

I replaced linting with plain npm scripts and that works well but has no "watch" functionality. The new solution can hopefully add a watch command to linting as well.

**Comment by vimal1083** [comment]
It is better to use webpack for all operations, working on the same. will let you know once it is done

**Comment by kominoshja** [comment]
If Gulp is no longer needed, it should probably be removed from https://docs.hypha.app/administrators/deployment/stand-alone/#install-python-packages, at least the change in `.circleci/config.yml` (replacing gulp with npm run build)

**Comment by frjo** [comment]
Thanks for alerting us to this. Fixed now.

P.S. Please open a new issue in the future.
